src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -89,6 +89,20 @@ internal IEnumerable<PackageHeader> ListAll()
             }, new List<PackageHeader>());
         }
 
+        /// <summary>
+        /// Make a call to Package Manager to get the known
+        /// supported hosts for package publishing and filtering
+        /// </summary>
+        internal IEnumerable<string> GetKnownHosts()
+        {
+            return FailFunc.TryExecute(() =>
+            {
+                var hosts = new Hosts();
+                var hostsResponse = this.client.ExecuteAndDeserializeWithContent<List<String>>(hosts);
+                return hostsResponse.content;
+            }, new List<string>());
```

Comment:
does this return an empty list if this request fails? How long until it times out?

Reply:
Yes, empty list if fails. Times out depends on the `HttpClient.Timeout` initialization in Greg, I think it is 100 sec without touching the `Timeoutspan` property.